odl/solvers/nonsmooth/douglas_rachford.py

```diff
@@ -28,7 +28,7 @@
 __all__ = ('douglas_rachford_pd',)
 
 
-def douglas_rachford_pd(x, prox_f, prox_cc_g, L, tau, sigma, niter,
+def douglas_rachford_pd(x, f, g, L, tau, sigma, niter,
```

Comment:
Decent suggestion for names, although id go for `inf_conv_funcs` for consistency.